docs/services/topology.md

```diff
@@ -73,7 +79,15 @@ HTTPD configuration is in `/etc/httpd`; we use the modules `mod_ssl`, `mod_grids
 The first two are installed via yum;
 the .so file for mod_wsgi is located in `/opt/topology/venv/lib/python3.6/site-packages/mod_wsgi/server/`.
 
-**TODO Derek can you write something here?**
+Each of the hostnames are VHosts in the apache configuration.  Some special notes:
+
+* <https://map.opensciencegrid.org> runs in the same wsgi process as the production topology, but the URL is limited to only the map code.  Further, it does not use mod_gridsite so that users are not asked to present a client certificate.
+* VHosts are configured:
+
+        :::console
+        ServerName topology.opensciencegrid.org
+        ServerAlias my.opensciencegrid.org myosg.opensciencegrid.org
+
```

Comment:
I take it that the ITB hosts are configured like so?

```
ServerName topology-itb.opensciencegrid.org
ServerAlias my-itb.opensciencegrid.org
```

Reply:
Yes.  I just wanted to give a sample of the config.

Reply:
I think we should include all of the necessary config here. One of the use cases I want to address with this document is "How do we spin up a new topology service if the main one goes down?"

Reply:
The total apache config for everything is on the order of ~100 lines.  Should I copy / paste everything in here?

Reply:
Are you saying that the topology apache service requires ~100 lines of config on top of the default apache config? I'm more interested in the diff

Reply:
I was curious, so I started `wc`'ing stuff.

* To just get the production topology working: 43 non-empty, non-comment lines.
* ITB instance: 26 lines
* Map: 28 lines.

So 97 lines just in the custom configuration.  My guess was surprisingly close!

Reply:
I'm ok with that. Throw the config in between details tags: https://github.com/opensciencegrid/docs/blame/master/docs/monitoring/install-rsv.md#L233-L234

Reply:
@djw8605 ^^